.github/workflows/chart-release.yaml

```diff
@@ -0,0 +1,58 @@
+name: Release charts
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Is this workflow supposed to run on every push to main, or should it run when we tag the repo for example?

Do you have an example of what the output should be? I've looked at https://github.com/helm/chart-releaser but could not understand immediately what we wanted to achieve.

Reply:
Helm chart and ORC itself will not have the same release cycle (I think), so link them is not practicable.
Keep as OCI is the simplest solution and my preferred one too

Reply:
While working on the OLM bundle, we decided it would be more practical to release the bundle image at the same time as ORC, each time we tag the repo. If we need to make changes to the manifest, it's easy enough to tag a new version.

Since the manifests are in the same repo, it also makes sense to inherit the same version number. I believe the same applies for helm charts.